app/src/main/res/values/strings.xml

```diff
@@ -135,4 +135,8 @@
     <string name="next_search_result">Next search result</string>
     <string name="help_no_results">No search results</string>
 
+    <string name="exposure_info_date">Naposledy %s jste se setkali s osobou u které bylo potvrzeno onemocnění COVID-19</string>
```

Comment:
Tady by to chtělo angličtinu, překlady budeš řešit až později?

Reply:
jj, vyřešil bych najednou